packages/playground/blueprints/src/lib/steps/common.ts

```diff
@@ -11,3 +11,12 @@ export function zipNameToHumanName(zipName: string) {
 		mixedCaseName.slice(1).toLowerCase()
 	);
 }
+
+type PatchFileCallback = (contents: string) => string | Uint8Array;
+export async function patchFile(
+	php: UniversalPHP,
+	path: string,
+	callback: PatchFileCallback
+) {
+	await php.writeFile(path, callback(await php.readFileAsText(path)));
```

Comment:
this seems like an odd mix of semantic and generality: it's called `patchFile` but it doesn't path files; it's more like `updateFile`, but then it it's not a generic file updated because it requires the `PathFileCallback`, but then that interface is more like `Mapper<string|Uint8Array>`

is it possible this is more `updateFile` or `mapFile`?

Reply:
You're right, it's more of a `updateFile`  👍  I'll update that and any other private copy&paste instances of this helper.

Reply:
Done in [a377bb0](https://github.com/WordPress/wordpress-playground/pull/214/commits/a377bb051ef5cfddf51511ab3e3d2434249ebcf6)